Makefile

```diff
@@ -6,8 +6,8 @@ CFLAGS_DEFAULT=	-O2
 
 all: cpusupport-config.h
 	export CFLAGS="$${CFLAGS:-${CFLAGS_DEFAULT}}";	\
-	export LDADD_POSIX=`export CC="${CC}"; cd libcperciva/POSIX && command -p sh posix-l.sh`;	\
-	export CFLAGS_POSIX=`export CC="${CC}"; cd libcperciva/POSIX && command -p sh posix-cflags.sh`;	\
+	export LDADD_POSIX=`export CC=\`which "${CC}"\`; cd libcperciva/POSIX && command -p sh posix-l.sh`;	\
```

Comment:
Passing it to `posix-l.sh`?  No.  Passing it to `posix-cflags.sh`?  Yes.  And since I did it for that, I figured that I might as well do it to the previous line.

Another option would be to pass in $PATH, but I figured that adjusting the ${CC} variable was a "smaller" change.  No?